code/modules/mining/abandoned_crates.dm

```diff
@@ -169,6 +169,7 @@
 				locked = FALSE
 				cut_overlays()
 				add_overlay("securecrateg")
+				tamperproof = 0
```

Comment:
would you mind putting a comment for why that's there so folks know at a glance what this memery is

Reply:
Sure